### DISCUSSION TITLE
Change thumbnail size in gallery page

### DISCUSSION THREAD

**Discussion by kintarowonders** [stephenmcd/mezzanine#2021] [Q&A]
I want to display some photos of my cat on my personal website. I decided to add the photos to a folder in the media library, and then create a gallery page with these images selected. This was pretty straightforward and I have a [gallery on my website of pictures of my cat](https://johntate.org/leto/) like I was expecting.

However the thumbnails on this page are far smaller than I would like. I could not find a setting for this within my admin panel. I don't mind editing source or doing whatever I have to do. I just want to be able to make the thumbnail images much larger than they currently are. I could not find any documentation for this on Google.

**Comment by skru** [comment]
You'll need to customise the page html. It is currently using the gallery.html, a little more on this: 

http://mezzanine.jupo.org/docs/frequently-asked-questions.html?highlight=copy%20template#where-are-all-the-templates-i-can-modify

So in your case you can modify pages/gallery.html which will change all gallery pages of your site or you could create a page: pages/leto.html and copy in the html from gallery.html

To modify the column/row count you'll need to read up on bootstrap: 

https://getbootstrap.com/docs/5.1/layout/columns/

To modify the thumbnails: 

http://mezzanine.jupo.org/docs/utilities.html#thumbnail

**Comment by kintarowonders** [comment]
Yes, I see. Thanks!

The Mezzanine core folders and also everything on my site is installed and managed by Pip (as a normal user with --user).

Is there a way to have certain aspects controlled outside Pip so I don't update over them?

I mean, I could use virtualenv and change things within pip but it would create a problem with updates to mezzaine and bootstrap. What do I do about this?

I might make a feature request because I was hoping there was just something I could add to my settings.py file.

**Comment by jerivas** [comment]
Mezzanine ships the html templates and a copy exists in the virtualenv / Python installation, but Django allows you to override templates by creating them in your project folder, just as @skru explained before. See https://docs.djangoproject.com/en/4.0/howto/overriding-templates/
